pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -462,6 +478,59 @@ def default_model_config(self) -> Dict:
             "gamma_control": {"dist": "Normal", "kwargs": {"mu": 0, "sigma": 2}},
             "gamma_fourier": {"dist": "Laplace", "kwargs": {"mu": 0, "b": 1}},
         }
+        if self.lagging_function == "geometric":
```

Comment:
so in my suggestion, all of these cases would belong to the class of "geometric", etc